app-doc/devmanual/devmanual-9999.ebuild

```diff
@@ -10,19 +10,30 @@ HOMEPAGE="https://devmanual.gentoo.org/"
 
 if [[ ${PV} == *9999 ]]; then
 	inherit git-r3
-	EGIT_REPO_URI="git://anongit.gentoo.org/proj/devmanual.git"
+	EGIT_REPO_URI="https://anongit.gentoo.org/git/proj/devmanual.git"
 else
 	SRC_URI="https://dev.gentoo.org/~hwoarang/distfiles/${P}.tar.gz"
 	KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~x64-macos"
 fi
 
 LICENSE="CC-BY-SA-2.0"
 SLOT="0"
-IUSE=""
+IUSE="fallback"
```

Comment:
I wonder if this shouldn't be enabled by default, or even made unconditional. Otherwise, what is the point of installing it locally if it still needs remote resources?

Reply:
How would I go about enabling it by default? I like that idea best, because it can still be turned off.

Reply:
With an IUSE default: https://devmanual.gentoo.org/ebuild-writing/eapi/index.html#eapi=1

Reply:
Ah! Thank you!